MapboxCoreNavigation/RouteController.swift

```diff
@@ -378,6 +378,7 @@ open class RouteController: NSObject {
     @objc public func suspendLocationUpdates() {
         locationManager.stopUpdatingLocation()
         locationManager.stopUpdatingHeading()
+        locationManager.delegate = nil
```

Comment:
@akitchen I had mentioned this recently, do we want to keep the `locationManager` alive for other intended purposes like _telemetry_ etc.?

Reply:
After deiniting, I don't think so.

Reply:
No, I was mainly saying that removing this object’s relationship to the location manager doesn’t necessarily power it down or reduce power consumption. Doing this in deinit is a good plan.

Reply:
Doing it here is fine too, if we’re not interested in receiving updates.